src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -163,6 +168,10 @@ function createFirstChild(returnFiber, existingChild, newChildren, priority) {
 
 // TODO: This API won't work because we'll need to transfer the side-effects of
 // unmounting children to the returnFiber.
-exports.reconcileChildFibers = function(returnFiber : Fiber, currentFirstChild : ?Fiber, newChildren : ReactNodeList, priority : PriorityLevel) : ?Fiber {
+exports.reconcileChildFibers = function(
+  returnFiber : Fiber, 
+  currentFirstChild : ?Fiber, 
+  newChildren : ReactNodeList, 
+  priority : PriorityLevel) : ?Fiber {
```

Comment:
My bad. Closing parenthesis should have been on new line. 
Can someone guide me on how can I push a patch?

We use Gerrit in house and all we need to do is to keep same change-id for a commit. That takes care of patching a commit already pushed.